File: java/01-commands/doc/sourcecode/src/main/java/com/training/app/BankingTransactionDTO.java
package com.training.app;

import java.time.LocalDate;

public class BankingTransactionDTO {
    private LocalDate date;
    private int amount;
    private int balance;

    public BankingTransactionDTO(LocalDate date, int amount, int balance) {
        this.date = date;
        this.amount = amount;
        this.balance = balance;
    }

    public LocalDate getDate() {
        return date;
    }

    public int getAmount() {
        return amount;
    }

    public int getBalance() {
        return balance;
    }
}

File: cpp/01-commands/doc/Banking_Kata/BankingTransactionDTO.cpp
#include "pch.h"
#include "BankingTransactionDTO.h"

BankingTransactionDTO::BankingTransactionDTO(timepoint date, long amount, long balance)
    : _date(date), _amount(amount), _balance(balance)
{}

const timepoint& BankingTransactionDTO::getDate() const
{
    return _date;
}

long BankingTransactionDTO::getAmount() const
{
    return _amount;
}

long BankingTransactionDTO::getBalance() const
{
    return _balance;
}
